urumanifest/commit.py

```diff
@@ -46,8 +54,13 @@ def _hash_asset(args: Tuple[Path, Path]) -> Tuple[Path, str, int]:
     server_path, source_path = args
     # One day, we will not use such a vulnerable hashing algo...
     h = hashlib.md5()
+    if source_path.is_dir():
+        bundle_name = source_path.stem
+        # Guess the executable path - we could get the exact name if we could unpack the Info.plist
+        # in a cross platform way.
+        source_path = source_path.joinpath(Path("Contents/MacOS/" + bundle_name))
     with source_path.open("rb") as in_stream:
-        _io_loop(in_stream, h.update)
+         _io_loop(in_stream, h.update)
```

Comment:
```suggestion
        _io_loop(in_stream, h.update)
```

Erroneous whitespace change?